brainstormforce/ultimate-addons-for-gutenberg
# Fix: Review Schema - Default underline issue for title from theme.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/3044d0ae-d1ca-49ba-b04e-d443a387ae07?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

 - Drag and drop Review schema and Title, description, author, and other details.
 - Check to underline for the title is visible in the editor as well as on the frontend. 
 - Also check only the text-decoration from typography should apply.

### Checklist:
